lib/tasks/krikri.rake

```diff
@@ -7,23 +7,36 @@ require 'resque/tasks'
 namespace :krikri do
 
   # Tasks will execute in spec/internal unless directory is changed
-  desc 'Index sample data in solr'
-  task :index_sample_data do
+  desc 'Index a valid sample aggregation'
+  task :index_sample_aggregation do
     agg = build(:aggregation)
 
     graph = agg.to_jsonld['@graph'][0]
 
     # prepend "krikri_sample" to @id so that sample data can be identified for
     # deletion
-    # TODO: add ids with prefixes (or similar identification) to factories
     graph['@id'] = 'krikri_sample' + graph['@id']
 
     Krikri::IndexService.add graph.to_json
     Krikri::IndexService.commit
   end
 
-  desc 'Delete sample data from solr'
-  task :delete_sample_data do
+  desc 'Index aggregations with missing required values'
+  task :index_invalid_aggregation do
+
+    agg = build(:aggregation,
+                :aggregatedCHO => build(:source_resource, title: nil)
```

Comment:
This line should be changed to `:sourceResource => build(:source_resource, title: nil)`, given recent changes to `dpla-map`.